tkweb/apps/drinks/views.py

```diff
@@ -0,0 +1,31 @@
+from django.http import HttpResponseRedirect
+from django.shortcuts import render, get_object_or_404
+from django.views.generic.edit import FormView
+
+from .forms import BarcardGenForm
+from .models import Barcard
+
+class BarcardSelect(FormView):
+    template_name = 'drinks/home.html'
+    form_class = BarcardGenForm
+    success_url = 'generate/'
+
+    def form_valid(self, form):
+        return super().form_valid(form)
```

Comment:
Lige for at være sikker: Har du skrevet noget kode så den her formular rent faktisk opdaterer hvilke drinks der er på drinkskortet? Det ser for mig ud som om at intet gemmes i databasen her i `form_valid()`?

Reply:
Det har jeg ikke nej. så burde jeg vel kunne undvære `form_valid(self, form)`?